src/components/user-card/UserCard.js

```diff
@@ -15,3 +15,24 @@ import React from 'react';
 //  этот div будет содержать:
 //  - блок h4 с классом card-title, он должен вывести имя и фамилию (first_name, last_name)
 //  - блок div c классом "card-text", содержащий 2 div блока для поля email и поля address
+
+export const UserCard  = (props) => {
+
+  const { user } = props;
+
+  if (!user) return null;
+
+  const { firstName, lastName, email, address, _links } = user;
+  return (
+    <div className="may-user-card card">
+      <img src={avatar} className="may-user-card-avatar rounded-circle" />
```

Comment:
Where `avatar` coming from? there would be an error
You need to get it from _links, and use avatar.href